Changes

```diff
@@ -32,7 +32,9 @@ be mentioned in the 4.06 section below instead of here.)
 
 ### Manual and documentation:
 
-### Compiler distribution build system:
+- PR#7647: emphasize ocaml.org website and forum in README (Yawar Amin)
```

Comment:
Sorry for the endless nitpicking, but:
- could you include `GPR#1384` as well?
- the credit line should be on its own line

Reply:
No worries, done and done :-)